### ISSUE TITLE
Force disable mods not working

### ISSUE THREAD

**Issue by HAHALOSAH** [LunarClient/BukkitImpl#12]
The Mods disabled notice shows up when I join but aren't actually disabled, I can still enable them in the mods menu

**Comment by hybridnetworkmc** [comment]
Same to me

**Comment by Fejm** [comment]
Same here @colinmcdonald22 @PrestigePvP

**Comment by hybridnetworkmc** [comment]
Its fixed now :)

**Comment by Fejm** [comment]
No, it's not. While you are making first connection it's working, just reconnect and boom, mods are not disabled even if notification appears.

**Comment by hybridnetworkmc** [comment]
Its worked before even i reconnect, but now doesn't work at all...
